packages/core/src/components/menu/context.ts

```diff
@@ -4,21 +4,12 @@
  * Licensed under the terms of the LICENSE file distributed with this project.
  */
 
-import { isFunction } from "../../common/utils";
+import { func } from "prop-types";
```

Comment:
not a fan of this import, the symbols are too terse. for example another possible import is `any`, which would be very confusing to import. please switch to `import * as PropTypes`

Reply:
yeah i felt the terseness. went with this for theoretical tree-shaking purposes, but i agree the `* as` import makes a lot more sense here.